app/controllers/guests_controller.rb

```diff
@@ -64,16 +67,45 @@ def set_guest
 
   def notify_host
     host = @guest.event.host
-    notification = GuestNotification.with(type: "join", host: host,
-                                          recipient: @guest.guest, status: @guest.status, event: @guest.event)
-    pp notification
+    notification = GuestNotification.with(
+      type: "event",
+      message: "#{@guest.guest.first_name} would like to join your event.",
+      recipient: @guest.guest,
```

Comment:
Just to clarify... recipient here shouldn't be the host? I know that the important part is in line 76 but would like to understand what recipient does here.

Reply:
recipient of the notification would be the host, but recipient in the params is the guest!